### PR TITLE
fix spelling mistake

### PR DESCRIPTION
**Pull request by oasangqi**
Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: etcdmain/gateway.go [hunk 1]
```diff
@@ -28,13 +28,12 @@ import (
 )
 
 var (
-	gatewayListenAddr            string
-	gatewayEndpoints             []string
-	gatewayDNSCluster            string
-	gatewayDNSClusterServiceName string
```
Comment (hexfusion) [inline etcdmain/gateway.go]: Why was this removed?

Comment (hexfusion) [reply etcdmain/gateway.go]: @oasangqi ping

**Comment by spzala** [comment]
@oasangqi following up @hexfusion 's earlier comment - are you still working on this PR? Thanks!

**Comment by jingyih** [comment]
Closing due to PR inactive.
